### DISCUSSION TITLE
[OP Stack Deployment] <Hi,I have deploy OP Stack,the bridge is not working.>

### DISCUSSION THREAD

**Discussion by xzsean666** [ethereum-optimism/developers#544] [Deploying the OP Stack]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

I do the same from the doc ,but its not working on my side.I send gas token to L1StandardBridgeProxy ,but I wait 1hour no gas token on l2.

anyway ,Why I want build this .because I need simulate the op evm,we're going to deploy dapps on op stack. but the test net eth is hard to get and the rpc is not stable for us.so Im trying deploy op stack L2 on bnb testnet .

is there any other way to do it? I just want build a local testnet of op stack,its better I can create gas token on my side.

someone help me the issue I meet ,thank you.

### Logs

op-geth

INFO [07-30|14:10:54.461] Gasprice oracle is ignoring threshold set threshold=2
WARN [07-30|14:10:54.462] Engine API enabled                       protocol=eth
INFO [07-30|14:10:54.462] Starting peer-to-peer node               instance=Geth/vuntagged-f2e69450-20240715/linux-amd64/go1.22.2
INFO [07-30|14:10:54.467] IPC endpoint opened                      url=/root/datadir/geth.ipc
INFO [07-30|14:10:54.468] Loaded JWT secret file                   path=jwt.txt crc32=0x840702e7
INFO [07-30|14:10:54.468] HTTP server started                      endpoint=[::]:8545 auth=false prefix= cors=* vhosts=*
INFO [07-30|14:10:54.468] WebSocket enabled                        url=ws://[::]:8546
INFO [07-30|14:10:54.468] WebSocket enabled                        url=ws://[::]:8551
INFO [07-30|14:10:54.468] HTTP server started                      endpoint=[::]:8551 auth=true  prefix= cors=localhost vhosts=*
INFO [07-30|14:10:54.468] New local node record                    seq=1,722,348,654,468 id=618935f69b5b1605 ip=127.0.0.1 udp=0 tcp=30303
INFO [07-30|14:10:54.468] Started P2P networking                   self="enode://0f302fa87475b10e18cfb7511e05d643806c17e2ad0a2fa6251803bebb6bb17174f167e9def139898e894661c780dc69939b45203b85cbfdfb11ca551f3b98bd@127.0.0.1:30303?discport=0"
INFO [07-30|14:10:54.540] Generated state snapshot                 accounts=2333 slots=2080 storage=392.14KiB dangling=0 elapsed=81.620ms
WARN [07-30|14:12:40.112] Served eth_getBlockByNumber              conn=[::1]:59994 reqid=3 duration="37.665µs" err="finalized block not found"
WARN [07-30|14:12:40.113] Served eth_getBlockByNumber              conn=[::1]:59994 reqid=4 duration="38.365µs" err="safe block not found"
INFO [07-30|14:17:53.119] Starting work on payload                 id=0x02f08ac1ba415e45
INFO [07-30|14:17:53.120] Updated payload                          id=0x02f08ac1ba415e45 number=1 hash=97ccc1..5f814a txs=1 withdrawals=0 gas=63965 fees=0 root=044a2f..c9d4b2 elapsed="480.227µs"
INFO [07-30|14:17:55.120] Stopping work on payload                 id=0x02f08ac1ba415e45 reason=timeout elapsed=2.000s
INFO [07-30|14:18:07.149] Imported new potential chain segment     number=1 hash=97ccc1..5f814a blocks=1 txs=1 mgas=0.064 elapsed=9.709ms     mgasps=6.588 age=46m46s snapdiffs=172.00B triedirty=0.00B
INFO [07-30|14:18:07.150] Chain head was updated                   number=1 hash=97ccc1..5f814a root=044a2f..c9d4b2 elapsed=1.050522ms  age=46m46s
INFO [07-30|14:18:07.153] Indexed transactions                     blocks=2 txs=1 tail=0 elapsed=1.982ms
INFO [07-30|14:25:21.773] Starting work on payload                 id=0x02976aecbf543d76
INFO [07-30|14:25:21.774] Updated payload                          id=0x02976aecbf543d76 number=2 hash=265426..fd2121 txs=1 withdrawals=0 gas=47653 fees=0 root=1cedf7..abf5d0 elapsed="331.279µs"
INFO [07-30|14:25:23.775] Stopping work on payload                 id=0x02976aecbf543d76 reason=timeout elapsed=2.001s
INFO [07-30|14:25:25.017] Imported new potential chain segment     number=2 hash=265426..fd2121 blocks=1 txs=1 mgas=0.048 elapsed=3.979ms     mgasps=11.975 age=54m2s  snapdiffs=453.00B triedirty=0.00B
INFO [07-30|14:25:25.019] Chain head was updated                   number=2 hash=265426..fd2121 root=1cedf7..abf5d0 elapsed=1.082903ms  age=54m2s
INFO [07-30|14:25:28.612] Starting work on payload                 id=0x02357caa541cfab9
INFO [07-30|14:25:28.612] Updated payload                          id=0x02357caa541cfab9 number=3 hash=d8efba..0dd87c txs=1 withdrawals=0 gas=49653 fees=0 root=8ed794..c080d4 elapsed="295.538µs"
INFO [07-30|14:25:28.612] Stopping work on payload                 id=0x02357caa541cfab9 reason=delivery elapsed="406.862µs"
INFO [07-30|14:25:28.616] Imported new potential chain segment     number=3 hash=d8efba..0dd87c blocks=1 txs=1 mgas=0.050 elapsed=3.348ms     mgasps=14.829 age=54m3s  snapdiffs=702.00B triedirty=0.00B
INFO [07-30|14:25:28.618] Chain head was updated                   number=3 hash=d8efba..0dd87c root=8ed794..c080d4 elapsed=1.016261ms  age=54m3s

----------------------------------------------------------------------------------------------------------------------
op-node:
RN [07-30|14:31:12.815] Served optimism_syncStatus               conn=[::1]:34646 reqid=168 duration=10.001035747s err="context canceled"
INFO [07-30|14:31:13.782] Advancing bq origin                      origin=a77f03..10b6f6:42542387 originBehind=false
WARN [07-30|14:31:15.359] L1 head signal indicates a possible L1 re-org old_l1_head=5d7692..93c08a:42543273 new_l1_head_parent=06bd95..4fc0f1 new_l1_head=b76c05..4a058d:42543277
INFO [07-30|14:31:17.300] Advancing bq origin                      origin=79c10d..f037c9:42542388 originBehind=false
INFO [07-30|14:31:19.784] Advancing bq origin                      origin=7e3e28..dad544:42542389 originBehind=false
INFO [07-30|14:31:21.985] Advancing bq origin                      origin=2c5b5a..7ad108:42542390 originBehind=false
WARN [07-30|14:31:23.756] L1 head signal indicates a possible L1 re-org old_l1_head=b76c05..4a058d:42543277 new_l1_head_parent=6d3511..4f1168 new_l1_head=eebf64..b9c892:42543281
INFO [07-30|14:31:25.413] Advancing bq origin                      origin=8edf2e..4750e4:42542391 originBehind=false
INFO [07-30|14:31:28.982] Advancing bq origin                      origin=953ab5..042bf6:42542392 originBehind=false
INFO [07-30|14:31:31.176] Advancing bq origin                      origin=858062..0d03ae:42542393 originBehind=false
WARN [07-30|14:31:34.879] Served optimism_syncStatus               conn=[::1]:36088 reqid=171 duration=10.0006622s   err="context canceled"
INFO [07-30|14:31:35.582] Advancing bq origin                      origin=0664d6..15ec3c:42542394 originBehind=false
WARN [07-30|14:31:38.110] L1 head signal indicates a possible L1 re-org old_l1_head=eebf64..b9c892:42543281 new_l1_head_parent=82d322..b9b103 new_l1_head=1c977d..09a9f8:42543286
INFO [07-30|14:31:38.634] Advancing bq origin                      origin=cc671f..86a694:42542395 originBehind=false
INFO [07-30|14:31:42.844] Advancing bq origin                      origin=544d9a..2d05c8:42542396 originBehind=false
INFO [07-30|14:31:45.053] Advancing bq origin                      origin=d643e1..7d0ade:42542397 originBehind=false
WARN [07-30|14:31:46.528] Served optimism_syncStatus               conn=[::1]:36072 reqid=89  duration=10.001019309s err="context canceled"
WARN [07-30|14:31:48.601] Served optimism_syncStatus               conn=[::1]:59536 reqid=173 duration=10.000974523s err="context canceled"
WARN [07-30|14:31:49.386] L1 head signal indicates a possible L1 re-org old_l1_head=1c977d..09a9f8:42543286 new_l1_head_parent=8621ca..ec9fb4 new_l1_head=3df77b..aa5cba:42543290
INFO [07-30|14:31:50.069] Advancing bq origin                      origin=e1fa61..e04f74:42542398 originBehind=false
INFO [07-30|14:31:52.845] New L1 finalized block                   l1_finalized=7926dc..d93257:42543287
INFO [07-30|14:31:52.845] received L1 finality signal, but missing data for immediate L2 finalization prev_finalized_l1=7926dc..d93257:42543287 signaled_finalized_l1=7926dc..d93257:42543287
INFO [07-30|14:31:52.845] New L1 safe block                        l1_safe=3df77b..aa5cba:42543290
INFO [07-30|14:31:53.373] Advancing bq origin                      origin=eb1825..0393a0:42542399 originBehind=false
INFO [07-30|14:31:56.118] Advancing bq origin                      origin=95d388..213f38:42542400 originBehind=false
WARN [07-30|14:31:59.086] Served optimism_syncStatus               conn=[::1]:60418 reqid=174 duration=9.999911616s  err="context canceled"
INFO [07-30|14:32:00.599] Advancing bq origin                      origin=c02296..e1a9fa:42542401 originBehind=false
WARN [07-30|14:32:02.755] L1 head signal indicates a possible L1 re-org old_l1_head=3df77b..aa5cba:42543290 new_l1_head_parent=a4ecab..225b6b new_l1_head=71e181..fa6970:42543295
INFO [07-30|14:32:03.328] Advancing bq origin                      origin=c27def..31ca6b:42542402 originBehind=false
INFO [07-30|14:32:06.338] Advancing bq origin                      origin=ccbebf..f3329f:42542403 originBehind=false
INFO [07-30|14:32:08.847] Advancing bq origin                      origin=38fd51..2b77ad:42542404 originBehind=false
WARN [07-30|14:32:10.436] Served optimism_syncStatus               conn=[::1]:60412 reqid=91  duration=10.000020693s err="context canceled"
INFO [07-30|14:32:11.680] Advancing bq origin                      origin=11a5ac..44e55b:42542405 originBehind=false
INFO [07-30|14:32:14.132] Advancing bq origin                      origin=5d80e9..862ed2:42542406 originBehind=false
WARN [07-30|14:32:16.025] L1 head signal indicates a possible L1 re-org old_l1_head=71e181..fa6970:42543295 new_l1_head_parent=7e5401..03b112 new_l1_head=9146be..3e7400:42543299
INFO [07-30|14:32:16.578] Advancing bq origin                      origin=f358dd..2bfaa0:42542407 originBehind=false

------------------------------------------------------------------------------------------------------------------------
batcher

INFO [07-30|14:18:20.168] added L2 block to local state            block=97ccc1..5f814a:1 tx_count=1 time=1,722,346,281
INFO [07-30|14:18:20.590] Created channel                          id=1848c0..c8b2aa l1Head=289cad..b3194e:42543021 blocks_pending=1 batch_type=0 max_frame_size=119,999
WARN [07-30|14:18:30.591] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:19:09.455] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:19:41.818] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:20:02.056] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:20:13.107] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:20:38.063] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:21:08.343] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:21:34.623] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:21:58.259] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:22:20.638] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:22:31.028] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:22:50.334] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:23:01.412] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:23:45.910] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
INFO [07-30|14:25:25.022] added L2 block to local state            block=265426..fd2121:2 tx_count=1 time=1,722,346,283
INFO [07-30|14:25:31.597] added L2 block to local state            block=d8efba..0dd87c:3 tx_count=1 time=1,722,346,285
WARN [07-30|14:26:28.086] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:26:47.021] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:27:03.121] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:27:13.589] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:27:24.060] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:27:53.652] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:28:16.078] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:29:11.995] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:29:22.476] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:29:42.926] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:30:18.960] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:30:58.498] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:31:12.815] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:31:34.879] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:31:48.601] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:31:59.086] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:32:38.582] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"
WARN [07-30|14:32:57.349] Error calculating L2 block range         err="failed to get sync status: Post \"http://localhost:8547\": context deadline exceeded"

------------------------------------------------------------------------------------------------------------------------------
proposer

INFO [07-30|14:13:46.470] metrics disabled 
INFO [07-30|14:13:47.999] Connected to L2OutputOracle              address=0x8a9e2337B0DD5bd8183edB07df3f6f260D025bEe version=1.7.0
INFO [07-30|14:13:48.004] Starting JSON-RPC server 
INFO [07-30|14:13:48.015] Starting Proposer 
INFO [07-30|14:13:48.015] Starting Proposer 
INFO [07-30|14:13:48.015] Proposer started 
ERROR[07-30|14:14:46.538] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:15:34.501] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:16:35.269] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:16:58.532] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:17:22.902] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:17:46.519] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:17:58.540] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:18:10.519] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:18:46.549] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:18:58.524] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:20:35.008] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:21:22.525] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:21:34.502] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:22:23.405] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:22:35.776] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:23:46.497] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:26:22.543] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:26:36.184] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:27:22.430] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:27:34.823] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:27:46.480] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:28:35.168] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:28:59.207] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:29:59.338] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:30:22.476] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:30:58.493] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:31:10.814] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:31:46.528] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:32:10.436] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:32:58.502] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"
ERROR[07-30|14:33:46.505] proposer unable to get sync status       err="Post \"http://localhost:8547\": context deadline exceeded"

**Comment by opfocus** [comment]
The error log appear that batcher and proposer cannot connect to op-node component . please check:
```
curl -X POST -H "Content-Type: application/json" --data \
    '{"jsonrpc":"2.0","method":"optimism_rollupConfig","params":[],"id":1}'  \
    http://localhost:8547
```

**Comment by opfocus** [comment]
The error log appear that batcher and proposer cannot connect to op-node component . please check:
```
curl -X POST -H "Content-Type: application/json" --data \
    '{"jsonrpc":"2.0","method":"optimism_syncStatus","params":[],"id":1}'  \
    http://localhost:8547
```
